src/components/WebCLI/WebCLI.tsx

```diff
@@ -159,10 +172,11 @@ const WebCLI = ({
         );
         authentication.user = activeUser ? getUserName(activeUser) : undefined;
         authentication.macaroons = [deserialized];
-      } else {
-        // XXX Surface error to the user.
-        console.error("No authentication information available");
       }
+      setInlineError(
+        InlineErrors.AUTHENTICATION,
+        macaroons ? null : Label.AUTHENTICATION_ERROR,
```

Comment:
While I don't think we should pop open the console if there is a connection error, I think we should open it when someone tries to send a command and then get this error.

Reply:
Makes sense, added this in my latest commit.